### PR TITLE
PKG-156 - Fix for update of telemetry-agent package failure

### PR DESCRIPTION
**Pull request by surbhat1595**

### CODE REVIEW

File: packaging/debian/postinst [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-chmod 775 /var/log/percona/telemetry-agent
-systemctl daemon-reload
-systemctl enable percona-telemetry-agent.service || true
-systemctl start percona-telemetry-agent.service
+# Create log directory
+install -d -m 0775 -o daemon -g percona-telemetry /var/log/percona/telemetry-agent
+install -D -m 0660  -o daemon -g percona-telemetry /dev/null /var/log/percona/telemetry-agent/telemetry-agent.log
+install -D -m 0660  -o daemon -g percona-telemetry /dev/null /var/log/percona/telemetry-agent/telemetry-agent-error.log
+# Move the old log-files into the new directory
+if [ -e /var/log/percona/telemetry-agent.log ] && [ -e /var/log/percona/telemetry-agent-error.log ]; then
```
Comment (maxkondr) [inline packaging/debian/postinst:9]: I would check each log file separately, I mean 2 `if` blocks

File: packaging/rpm/telemetry-agent.spec [hunk 2]
```diff
@@ -73,42 +76,60 @@ if [ ! -d /run/percona-telemetry-agent ]; then
...
 %post -n percona-telemetry-agent
-%systemd_post percona-telemetry-agent.service
-/usr/bin/systemctl enable percona-telemetry-agent >/dev/null 2>&1 || :
-/usr/bin/systemctl start percona-telemetry-agent
+chown -R daemon:percona-telemetry %{_log_dir} >/dev/null 2>&1 || :
+chmod g+w %{_log_dir}
+# Move the old logfiles, if present during update
+if [ -e /var/log/percona/telemetry-agent.log ] && [ -e /var/log/percona/telemetry-agent-error.log ]; then
+    chmod 0775  %{_log_dir}
+    mv /var/log/percona/telemetry-agent.log /var/log/percona/telemetry-agent/
```
Comment (maxkondr) [inline packaging/rpm/telemetry-agent.spec:89]: btw, what with the rest of logs that were already rotated and have suffixes like `.log.1` ?

File: packaging/debian/postinst (hunk 1 above)
Comment (surbhat1595) [reply packaging/debian/postinst]: I have changed the "&&" condition to "||", so it's evaluated even if one of the log file is present

File: packaging/rpm/telemetry-agent.spec (hunk 2 above)
Comment (surbhat1595) [reply packaging/rpm/telemetry-agent.spec]: Yes, good point, thanks. I've updated the changes to move the logrotated files as well.

**Review by maxkondr — APPROVED** [review]

**Pull request merged**